OpenMarket/OpenMarket/Network/URLSessionGenerator.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  URLSessionProvider.swift
+//  OpenMarket
+//
+//  Created by Grumpy, OneTool on 2022/05/10.
+//
+
+import Foundation
+
+class URLSessionGenerator {
+    let session: CustomURLSession
+    let apiHost: String = "https://market-training.yagom-academy.kr/"
+    
+    init (session: CustomURLSession = URLSession.shared) {
+        self.session = session
+    }
+    
+    func sendGet(path: String, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void){
```

Comment:
네트워크 네이밍 관련해서 애플 문서나 [alamofire](https://github.com/Alamofire/Alamofire) 라이브러리를 많이 참고하는 것 같아요.
sendGet보다 request라는 메서드명을 추천드립니다.
```suggestion
    func request(path: String, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void){
```